x/posts/types/models/reactions/post_reaction.go

```diff
@@ -32,7 +32,7 @@ func NewPostReaction(shortcode, value string, owner sdk.AccAddress) PostReaction
 
 // String implements fmt.Stringer
 func (reaction PostReaction) String() string {
-	out := fmt.Sprintf("[Shortcode] %s [Value] %s [Owner] %s", reaction.Shortcode, reaction.Value, reaction.Owner)
+	out := fmt.Sprintf("[Shortcode] %s [Value] %s [CurrentOwner] %s", reaction.Shortcode, reaction.Value, reaction.Owner)
```

Comment:
I think it's better to leave this as `Owner` instead. I don't see how a post reaction could change owner over time. 

```suggestion
	out := fmt.Sprintf("[Shortcode] %s [Value] %s [Owner] %s", reaction.Shortcode, reaction.Value, reaction.Owner)
```

Reply:
refactoring error